NBitcoin.Tests/AltcoinTests.cs

```diff
@@ -252,8 +312,16 @@ private void CheckCapabilities(RPCClient rpc, string command, bool supported)
 		[Fact]
 		public void CanSyncWithPoW()
 		{
+
 			using (var builder = NodeBuilderEx.Create())
 			{
+				if (new[] {Liquid.Instance.Mainnet, Liquid.Instance.Testnet, Liquid.Instance.Regtest,}.Contains(
```

Comment:
Please use a method for this copy/paste.

Reply:
haha yes